exercises/exercise3.js

```diff
@@ -1,12 +1,102 @@
 // Set this assignment to true if you do want to use it.
-module.exports.ACTIVATE_BOT = false;
+module.exports.ACTIVATE_BOT = true;
 
 module.exports.botScripts = [
   {
-    label: 'An Example Script',
-    prompt: 'Good Evening',
+    label: 'Greeting',
+    prompt: 'Hello',
     handler: function () {
-      return 'Good Evening to You!';
+      return 'Hello, welcome to JP\'s baseball info bot. Please enter your name to get started.';
     },
+    isListening: true,
+  },
+  {
+    label: 'Name Entry',
+    prompt: 'My name is #{firstName}',
+    handler: function (params) {
+      return 'Thank you ' + params.firstName + ' how can I help you today?';
+    },
+    isListening: true,
+  },
+  {
+    label: 'Batting Average Calculation',
+    prompt: 'If I have #{seasonHits} hits and #{seasonAtBats} at bats what is my batting average?',
+    handler: function (params) {
+      var battingAverage = params.seasonHits / params.seasonAtBats;
+      return 'Your batting average is ' + battingAverage;
```

Comment:
Love this! Though as a point of order, I mostly see batting averages limited to three decimal points in player's stats...might try 'battingAverage.toFixed(3)'.